### PR TITLE
cleaning python wrappers

### PR DESCRIPTION
**Pull request by dalg24**

### CODE REVIEW

File: python/source/wrappers/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+SET(PyCap_HEADERS
+   ${CMAKE_CURRENT_SOURCE_DIR}/property_tree_wrappers.h
+   ${CMAKE_CURRENT_SOURCE_DIR}/energy_storage_device_wrappers.h
+)
+SET(PyCap_SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/property_tree_wrappers.cc
+#    ${CMAKE_CURRENT_SOURCE_DIR}/energy_storage_device_wrappers.cc
```
Comment (Rombur) [inline python/source/wrappers/CMakeLists.txt:8]: Don't comment the code inside the set. It's just confusing for other people.

**Comment by Rombur** [comment]
If the tests pass, just fix the comment, and merge yourself.

File: python/source/wrappers/CMakeLists.txt (hunk 1 above)
Comment (dalg24) [inline python/source/wrappers/CMakeLists.txt:8]: Ok I will add a comment. Please look at python/source/wrappers/export_energy_storage_device.cc

I had to include the .cc files in it

Comment (Rombur) [inline python/source/wrappers/CMakeLists.txt:8]: I know but in three months, I will see that line and wondered why it is commented and I will uncomment/delete it :-)

**Pull request merged**